src/Auth/AuthServiceProvider.php

```diff
@@ -15,14 +15,14 @@ public function boot(Dispatcher $dispatcher): void
         $dispatcher->listen(RegisterDirectiveNamespaces::class, static fn (): string => __NAMESPACE__);
     }
 
-    public static function guard(): ?string
+    /**
+     * @return array<string>
+     */
+    public static function guards(): array
     {
         $config = Container::getInstance()->make(ConfigRepository::class);
-        $lighthouseGuard = $config->get('lighthouse.guard');
-        $guards = $config->get('auth.guards');
 
-        return isset($guards[$lighthouseGuard])
-            ? $lighthouseGuard
-            : $config->get('auth.defaults.guard');
+        return $config->get('lighthouse.guards')
+            ?? [$config->get('auth.default.guard')];
```

Comment:
```suggestion
            ?? [$config->get('auth.defaults.guard')];
```

Reply:
I was first surprised existing unit tests was passing with this typo, but in this end, this resulted with a null guard that behaves the same as if it was the default guard.